pos-print/src/main/java/com/clouway/pos/print/adapter/http/HttpModule.java

```diff
@@ -22,6 +22,7 @@ protected void configureSitebricks() {
         at("/_status").serve(StatusService.class);
         at("/v1/posprint").serve(PrintService.class);
         at("/v1/devices").serve(CashRegistersService.class);
+        at("/v1/report").serve(ReportService.class);
```

Comment:
Maybe it would be better services to be named as follow:

/v1/print/receipt (fiscal, text)
/v1/print/reports/operator (print operator report)
/v1/print/reports/periodical (print periodical report) 

cc @mlesikov @martinmilev